app/views/admin/words/index.html.erb

```diff
@@ -18,7 +18,7 @@
                 <% @words.each do |word| %>
                 <tr>
                     <td><%= word.content %></td>
-                    <td><%= word.correct_ans%></td>
+                    <td><%= word.correct_ans.content%></td>
```

Comment:
@johnedelmapa This change is actually same with the implementation before.
The error is occurring only on first category regarding on the data that is related with the `word` object. Please investigate more on that.

Reply:
@johnedelmapa Yes, as you can see my comment before is not resolved yet

Reply:
everything works perfectly on my side, no errors. I have no idea whats the error, but I added some line of codes please check. please run rails db:migrate:reset first

Reply:
and db:seed